src/app/clients/edit-client/edit-client.component.html

```diff
@@ -134,10 +146,6 @@
             </mat-select>
           </mat-form-field>
 
-          <mat-checkbox *ngIf="editClientForm.value.legalFormId === 1" fxFlex="48%" labelPosition="before" formControlName="isStaff" class="margin-v">
```

Comment:
Has this been moved above? If so, why?

Reply:
It is just to set a better view distribution, I mean set the contact data, mobile and email in the same line